code/modules/mob/dead/observer/observer.dm

```diff
@@ -54,7 +54,6 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	if(ismob(body))
 		T = get_turf(body)				//Where is the body located?
 		attack_log_old = body.attack_log_old	//preserve our attack logs by copying them to our ghost
```

Comment:
shouldn't this be a `Copy` function as well?

Reply:
This is the old way of doing it. Ghosts get the reference of the attack logs of the mob they belong to. I've left the old way in for legacy sake but I'm willing to fully remove it if admins like the new system enough